tests/lib/radar.js

```diff
@@ -98,6 +99,9 @@ Service.start = function(configuration, callback) {
   configuration.sentry = assertHelper.SentryDefaults;
 
   radar = new Radar.server();
+  
+  radar.use(new QuotaManager());
```

Comment:
The Quota management behavior will be optional, so it will not be enabled by default.